src/sinks/datadog/logs.rs

```diff
@@ -231,6 +232,12 @@ impl HttpSink for DatadogLogsJsonService {
 
     async fn build_request(&self, events: Self::Output) -> crate::Result<http::Request<Vec<u8>>> {
         let body = serde_json::to_vec(&events)?;
+        if !events.is_empty() {
```

Comment:
Should this be `if !body.is_empty()`?

Reply:
In this case we will report healthcheck as well, which I guess it not what we want. I will probably add a comment to make it clear.

Reply:
Added a comment, please let me know if it is unclear/you disagree.

Reply:
Good enough.